src/main/frontend/components/NavBar.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import { Switch, Route, Link, Redirect } from "react-router-dom"
+
+import ConcertVenueForm from "./ConcertVenueForm"
+import VenueIndex from "./VenueIndex.js"
+
+const NavBar = props => {
+
+
+
+return(
+<div>
+      <Switch>
+        <Route exact path="/">
+          <Redirect to="/concert-venues" />
+        </Route>
+        <Route exact path="/concert-venues/new" component={ConcertVenueForm} />
```

Comment:
per comment about the form, this line should also be removed from this PR and added to your next one